credentials/credentials_test.go

```diff
@@ -55,18 +55,41 @@ func TestTLSClone(t *testing.T) {
 type serverHandshake func(net.Conn) (AuthInfo, error)
 
 func TestClientHandshakeReturnsAuthInfo(t *testing.T) {
-	done := make(chan AuthInfo, 1)
-	lis := launchServer(t, tlsServerHandshake, done)
-	defer lis.Close()
-	lisAddr := lis.Addr().String()
-	clientAuthInfo := clientHandle(t, gRPCClientHandshake, lisAddr)
-	// wait until server sends serverAuthInfo or fails.
-	serverAuthInfo, ok := <-done
-	if !ok {
-		t.Fatalf("Error at server-side")
+	tcs := []struct {
+		name string
+		address string
+	} {
+		{
+			name: "localhost",
+			address: "localhost:0",
+		},
+		{
+			name: "ipv4",
+			address: "127.0.0.1:0",
+		},
+		// Re-enable when TravisCI supports IPv6
+		//{
+		//	name: "ipv6",
+		//	address: "[::1]:0",
+		//},
```

Comment:
Instead of all this, I'd prefer something like "// TODO: test address "[::1]:0" when IPv6 is supported".

Among other things, we may not be on Travis forever.

Reply:
Removed the comment and went with the suggestion from https://github.com/grpc/grpc-go/pull/3082#issuecomment-539743979. Happy to adjust if you prefer a different approach to IPv6 coverage.